.ci/scripts/yamllint

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -e
+docker run --rm -v "$PWD:/yaml" sdesbure/yamllint yamllint "$@"
```

Comment:
is this image trustworthy?

Reply:
yamllint it's also widely used, but this image, comes from a user, so for both images we will make our own trustworthy image based on their Dockerfiles